src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/service/impl/FeatureTogglesServiceImpl.java

```diff
@@ -49,7 +47,7 @@ public class FeatureTogglesServiceImpl implements FeatureTogglesService {
     @Override
     public CompletableFuture<FeatureTogglesDto> toggleFeatures(FeatureTogglesDto dto) {
         String correlationId = RequestCorrelationId.getId();
-        logger.debug("Processing request to toggle features, new properties are {}", dto);
+        log.info("API request: Toggle features, new properties are {}", dto);
```

Comment:
Let's point that it's update request. Something like "Updating feature toggles. ***"